### PR TITLE
reduce create account steps in react.js client

### PR DESCRIPTION
**Pull request by mxfactorial**
account created from landingScreen
unit tests
e2e tests

### CODE REVIEW

File: react/src/components/AuthForm/DateInput/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  state = {
+    type: 'text'
+  }
+  handleFocus = () => {
+    this.setState({ type: 'date' })
+  }
+
+  handleBlur = () => {
+    this.setState({ type: 'text' })
+  }
```
Comment (oozywaters) [inline react/src/components/AuthForm/DateInput/index.js:14]: Why change input type?

File: react/src/components/AuthForm/index.js [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+
+  static getDerivedStateFromProps(nextProps, prevState) {
+    const { values } = nextProps
+    const { focused } = prevState
+    return values && !focused ? { values } : null
+  }
+
+  componentDidMount() {
+    if (this.props.submitOnEnter) {
+      document.addEventListener('keydown', this.listenEnterKeyDown)
```
Comment (oozywaters) [inline react/src/components/AuthForm/index.js]: Can wrap input fields in `<form>` tag to submit automatically

**Review by oozywaters — APPROVED** [review]

File: react/src/components/AuthForm/DateInput/index.js (hunk 1 above)
Comment (mxfactorial) [reply react/src/components/AuthForm/DateInput/index.js:14]: cosmetic. 

dateType

avoids date display onblur

textType

File: react/src/components/AuthForm/index.js (hunk 2 above)
Comment (mxfactorial) [reply react/src/components/AuthForm/index.js]: change added in https://github.com/systemaccounting/mxfactorial/pull/123/commits/f6f43c882d351e36f6508cf8231048fd209b922f

**Pull request merged**